### ISSUE TITLE
input_type = mimetypes.guess_type(args.input)[0].split('/')[0]

### ISSUE THREAD

**Issue by sidh1603** [open-mmlab/mmpose#2267]
Hi,

after cloning mmpose , i run that command 

python demo/topdown_demo_with_mmdet.py \
    projects/rtmpose/rtmdet/person/rtmdet_nano_320-8xb32_coco-person.py \
    https://download.openmmlab.com/mmpose/v1/projects/rtmpose/rtmdet_nano_8xb32-100e_coco-obj365-person-05d8511e.pth \
    projects/rtmpose/rtmpose/body_2d_keypoint/rtmpose-m_8xb256-420e_coco-256x192.py \
    https://download.openmmlab.com/mmpose/v1/projects/rtmposev1/rtmpose-m_simcc-aic-coco_pt-aic-coco_420e-256x192-63eb25f7_20230126.pth \
    --input webcam \
    --show

i got this error
input_type = mimetypes.guess_type(args.input)[0].split('/')[0]
![Screenshot from 2023-04-20 10-34-28](https://user-images.githubusercontent.com/51013394/233417725-5468bd08-3c2e-4ba6-998f-3e2bcdfa2d9b.png)

**Comment by Ben-Louis** [comment]
Using webcam input in `demo/topdown_demo_with_mmdet.py` is only supported in `dev-1.x` branch now.

**Comment by sidh1603** [comment]
![Screenshot from 2023-04-20 11-01-21](https://user-images.githubusercontent.com/51013394/233423085-0de4ce4b-4c3e-4167-8848-53d1e132bc91.png)
![Screenshot from 2023-04-20 11-00-00](https://user-images.githubusercontent.com/51013394/233423086-02bf6ef8-ce18-4d93-b6d5-27a67020183e.png)


running this command in cd ~/Documents/open_lab/mmdeploy-1.0.0-linux-x86_64-cxx11abi/example/python$ 

i m getting error , for the same i have attached the screenshots

**Comment by Tau-J** [comment]
Could you try switching to `dev-1.x` branch? 

```shell
# in MMPose root
git checkout dev-1.x

python demo/topdown_demo_with_mmdet.py ......

```

**Comment by sidh1603** [comment]
yes,from dev-1.x i have installed it and run the commands aa per the documentationSent from my iPhoneOn Apr 21, 2023, at 12:04 AM, Tau ***@***.***> wrote:﻿
Could you try switching to dev-1.x branch?
# in MMPose root
git checkout dev-1.x

python demo/topdown_demo_with_mmdet.py ......

**Comment by Tau-J** [comment]
![](https://user-images.githubusercontent.com/51013394/233417725-5468bd08-3c2e-4ba6-998f-3e2bcdfa2d9b.png)

according to this sceenshot, your code is not the latest `dev-1.x`, maybe you need to pull again

**Comment by sidh1603** [comment]
what i have noticed that its working fine on window , but its creating problem on ubuntu.Sent from my iPhoneOn Apr 21, 2023, at 10:21 PM, Tau ***@***.***> wrote:﻿
Closed #2267 as completed.
